core/modules/arp_responder.h

```diff
@@ -0,0 +1,42 @@
+#ifndef BESS_MODULES_ARP_RESPONDER_H_
+#define BESS_MODULES_ARP_RESPONDER_H_
+
+#include <map>
+
+#include "../module.h"
+#include "../module_msg.pb.h"
+#include "../utils/endian.h"
+#include "../utils/ether.h"
+#include "../utils/ip.h"
+
+using bess::utils::Ethernet;
+using bess::utils::be32_t;
+
+// ARP cache entry struct which keeps mapping between IP and MAC
+struct arp_entry {
+  Ethernet::Address mac_addr;
+  be32_t ip_addr;
+  // timestamp used to expire cache entries
```

Comment:
Minor: since the other fields don't have comments, perhaps move the comment to the same line as `time`.